libraries/webInterface/gui/index.js

```diff
@@ -1429,88 +1429,85 @@ realityServer.gotClick = function (event) {
 
         let textContent = document.getElementById('textEntryObject').querySelector('.textfield').innerText;
 
-        if (textContent === 'Enter Name') {
+        if (!isNameOk(textContent)) {
             return;
-        } else {
-            console.log(textContent);
-            let removeNode = document.getElementById('textEntryObject');
-            realityServer.removeAnimated(removeNode);
+        }
 
-            if (textContent !== '') {
+        console.log(textContent);
+        let removeNode = document.getElementById('textEntryObject');
+        realityServer.removeAnimated(removeNode);
 
-                let objectName = textContent;
-                if (shouldAddWorldObject) {
-                    objectName = '_WORLD_' + textContent;
-                }
+        let objectName = textContent;
+        if (shouldAddWorldObject) {
+            objectName = '_WORLD_' + textContent;
+        }
 
-                // TODO: sanitize object names before creating, for example prevent _WORLD_local (github issue #20)
+        // TODO: sanitize object names before creating, for example prevent _WORLD_local (github issue #20)
```

Comment:
This is handled by isNameOk as far as I can see

Reply:
ah yes, being able to not create non-world objects with the same name as world objects is a non-intended consequence of this